prometheus.lua

```diff
@@ -95,6 +95,26 @@ function Counter:inc(value, label_values)
   self.prometheus:inc(self.name, self.label_names, label_values, value or 1)
 end
 
+local Gauge = Metric:new()
+-- Set a given gauge to `value`
+--
+-- Args:
+--   value: (number) a value to set the gauge to. Should be defined.
+--   label_values: an array of label values. Can be nil (i.e. not defined) for
+--     metrics that have no labels.
+function Gauge:set(value, label_values)
+	if value == nil then
```

Comment:
Could you please reindent this function to be consistent with the rest of the file (2 spaces instead of tabs, as declared by the `vim:` comment on the first line)?